src/Components/Pest/Installer.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * This file is part of Laravel Zero.
+ *
+ * (c) Nuno Maduro <enunomaduro@gmail.com>
+ *
+ *  For the full copyright and license information, please view the LICENSE
+ *  file that was distributed with this source code.
+ */
+
+namespace LaravelZero\Framework\Components\Pest;
+
+use LaravelZero\Framework\Components\AbstractInstaller;
+
+/** @internal */
+final class Installer extends AbstractInstaller
+{
+    /** {@inheritdoc} */
+    protected $name = 'install:pest';
+
+    /** {@inheritdoc} */
+    protected $description = 'Pest: Use the Pest testing framework for your test suite';
+
+    /** {@inheritdoc} */
+    public function install(): void
+    {
+        $this->remove('phpunit/phpunit', true);
+        $this->require('pestphp/pest', true);
```

Comment:
Would be cool to also replace the entire test suite by a pest one.

Reply:
That would be cool, but not sure it'd be possible if someone is using the Pest component on an existing app.

Reply:
Well, you can just ask: "Wish to replace the current test suite by a ..."?

Reply:
How about what I've just pushed (9cfa4102810196d96a2a211316a313b32818147a)?

Reply:
Sorry, @nunomaduro. Do you think this seems ok now? 🤔